### DISCUSSION TITLE
Markdown spell check

### DISCUSSION THREAD

**Discussion by radhikanarayanan22** [Milkdown/milkdown#317] [Q&A]
I'd like to when the spell check option in Milkdown will be available. 

Thanks,
Radhika

**Comment by Saul-Mirone** [comment]
The spell check is supported by browser through: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck

**Reply by rajatkulkarni95** [reply]
@Saul-Mirone Bumping this up, any way to disable spellcheck? Doing it for the 
`<ReactEditor editor={editor} spellCheck="false" />` doesn't do the deed (TS also doesn't like it)
